src/reporting-flow/sub-components/next-steps-step.tsx

```diff
@@ -12,19 +15,40 @@ interface Props {
 export function NextStepsStep( { stepNumber }: Props ) {
 	const relevantTaskIds = useAppSelector( selectRelevantTaskIds );
 	const allTasksAreComplete = useAppSelector( selectAllTasksAreComplete );
+	const issueFeatureId = useAppSelector( selectIssueFeatureId );
+	const issueType = useAppSelector( selectIssueType );
+	const activeStep = useAppSelector( selectActiveStep );
 
 	// It's a no-op since we don't use the edit button in this step!
 	const onEdit = useCallback( () => {
 		return;
 	}, [] );
 
 	const tasksExist = relevantTaskIds.length > 0;
+	const requiredInfoIsMissing =
+		( issueFeatureId === null || issueType === 'unset' ) && activeStep === 'nextSteps';
+	const noTasksAreConfigured = ! tasksExist && activeStep === 'nextSteps';
 
 	const isComplete = tasksExist && allTasksAreComplete;
 
 	let stepContentDisplay: ReactNode;
 	if ( tasksExist ) {
 		stepContentDisplay = <NextSteps />;
+	} else if ( requiredInfoIsMissing ) {
+		stepContentDisplay = (
+			<p className={ styles.badStateMessage } role="alert">
+				Hmm... We seem to be missing some information. This almost always means we could not parse
+				the info in the URL. Restart by clicking on the Bugomattic header at the top of the page.
+			</p>
+		);
+	} else if ( noTasksAreConfigured ) {
+		stepContentDisplay = (
+			<p className={ styles.badStateMessage } role="alert">
+				Hmm... It appears this feature area has no issue reporting configuration. You can try
+				reaching out to the team contacts listed below (if available). You can also ping the
```

Comment:
**suggestion (non-blocking):** Disclaimer: I'm not an expert on copy. I'm wondering if this message could be simplified a bit, so it avoids using the `if available` phrase. Perhaps by having a simple message with a contact list that can be added to. Something like:

> "Hmm, it appears this feature area has no issue reporting configuration. You can try reaching out to the contacts listed below for help:"
> - Bugomattic administrators 

If team contacts are available, then they would be added to the list:
> "Hmm, it appears this feature area has no issue reporting configuration. You can try reaching out to the contacts listed below for help:"
> - Example person
> - Example team
> - Bugomattic administrators 

Not sure if that adds more complexity, but just wanted to throw it out there!

Reply:
Ah, that's a good point!!

So, this actually brings up two ideas for me...
1. This is an error state, so we should log an error here, and tell them we are following up on it. We need to correct these bad configs ASAP!
2. I don't feel _super_ great about trying to intelligently parse the More Info for the best contacts, because there's a bit too much flexibility there and it might get complicated and not always pick the right ones. 🤔 But, there will almost always be _something_ in the More Info, so I think we can just say "contact the team" and defer to the judgment of the user. 

So, I'm thinking...

> Hmm, it appears this feature area has no issue reporting configuration. We've been notified and will correct this soon! In the meantime, you can try contacting the product team directly.

What do you think of that?

Reply:
> This is an error state, so we should log an error here, and tell them we are following up on it. We need to correct these bad configs ASAP!

Great point!

> I don't feel super great about trying to intelligently parse the More Info for the best contacts, because there's a bit too much flexibility there and it might get complicated and not always pick the right ones. 🤔 But, there will almost always be something in the More Info, so I think we can just say "contact the team" and defer to the judgment of the user.

Very true! Yeah, that's probably a good call. Just to clarify, is the More Info panel supposed to be visible during the error state? I didn't see it during my testing. 

> Hmm, it appears this feature area has no issue reporting configuration. We've been notified and will correct this soon! In the meantime, you can try contacting the product team directly.

Perfect! I really like that it mentions that we're notified, so the user doesn't feel completely on their own 😆